onnxruntime/__init__.py

```diff
@@ -19,6 +19,11 @@
 # here we need to save the exception and continue with Cuda version validation in order to post
 # meaningful messages to the user.
 # the saved exception is raised after device version validation.
+try:
+    import torch
+except ImportError:
+    pass
```

Comment:
Silently ignore an exception isn't a good idea.

Reply:
Torch is required for eager mode, but it is not necessary for general cases of onnxruntime. So, I can print a statement mentioning torch is not found. Or should I do something more?